### DISCUSSION TITLE
Tienen un discord o un grupo de wa?

### DISCUSSION THREAD

**Discussion by DaveOval** [Launch-X-Latam/MisionFrontEnd#2] [Q&A] [answered]
Que onda gente para preguntar si tienen un grupo para estar mas unidos

**Comment by DiegoIrribarren** [comment]
pues.. oficialmente creo que no, pero si crean uno, avisen

**Reply by MarioEduardoAlvarezTorres** [reply]
Esta prohibido

**Comment by pierowebdesign** [comment]
Tengo entendido que eso esta totalmente prohibido.. solo los live oficiales del Launch X

**Comment by Emilio17MC** [comment, marked as answer]
No, no está permitido realizar canales de comunicación externos.

**Comment by ProDeveloperRoy** [comment]
No esta permitido por politicas del programa.

**Comment by alexrdzuwu** [comment]
Donde puedo ingresar a las clases en vivo? No me dejo unirme a git por exceso de solicitudes y en Teams no me llega nada :( ya mandé msj en Instagram y no me responden :c

**Reply by OATS74** [reply]
Me pasa igual no veo la sección de hoy donde las puedo ver. Gracias

**Reply by JosafatJimenezB** [reply]
para cuando logren ingresar solo chequen esta info #22

**Reply by DonQuike** [reply]
Revisa el correo que te dieron de innovacion, creí que me llegaría a mi correo personal pero parece que todo va a llegar en aquel correo

**Comment by Ignacioskm** [comment]
Amigo, esta prohibido hacer grupos de Wsp, discord , telegram o cualquier red social. Lo mencionaron el primer día , saludos.

**Comment by JorgeDevmm** [comment]
Esta prohibido bro

**Comment by Mauricio-Mizhquiri** [comment]
Esta prohibido hacer ese tipo de cosas, y también aprendan a leer ahi ya se les dijo
